docs/app/pages/components/sections/date-time-picker/time-picker/snippets/app.html

```diff
@@ -0,0 +1,86 @@
+<timepicker [ngModel]="myTime" [min]="minTime" [max]="maxTime" [hourStep]="hstep" [minuteStep]="mstep" [showMeridian]="ismeridian"
+    [showSeconds]="showSec" [readonlyInput]="!isEnabled" [mousewheel]="mousewheel" [arrowkeys]="arrowkeys" (ngModelChange)="changed($event)">
+</timepicker>
+
+<p class="m-t">Time is: <em>{{myTime}}</em></p>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-group heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="minHour">min hour</label>
+                            <ux-number-picker [(value)]="minHour" min="0" max="23" (valueChange)="minTime.setHours($event)"></ux-number-picker>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="maxHour">max hour</label>
+                            <ux-number-picker [(value)]="maxHour" min="0" max="23" (valueChange)="maxTime.setHours($event)"></ux-number-picker>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="minMinute">min minute</label>
+                            <ux-number-picker [(value)]="minMinute" min="0" max="59" (valueChange)="minTime.setMinutes($event)"></ux-number-picker>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="maxMinute">max minute</label>
+                            <ux-number-picker [(value)]="maxMinute" min="0" max="59" (valueChange)="maxTime.setMinutes($event)"></ux-number-picker>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="row uxd-customize-row">
+                    <div class="col-md-6 col-sm-6">
+                        <div>
+                            <label for="hstep">hours step</label>
```

Comment:
Please use the Bootstrap property names, as I mentioned yesterday. This would be "hourStep".